### DISCUSSION TITLE
How to change size of a cylinder over time

### DISCUSSION THREAD

**Discussion by codingaudrey** [enable3d/enable3d#252] [Help / Questions] [answered]
I'm trying to increase a cylinder's radius over time, but I don't know how to go about this.
I've tried `this.object.geometry.parameters.radiusBody = 10` in the `update()` section of my code, but while doesn't change what is on screen.

**Comment by yandeu** [comment, marked as answer]
It is not possible to update a physics body. But you can remove the physics body, update the shape and re-create a new physics body. (But this is slow).

If your cylinder doesn't have a physics body, please check the threejs forum.
